ignite/contrib/handlers/param_scheduler.py

```diff
@@ -284,3 +284,165 @@ def __call__(self, engine, name=None):
                 self._next_scheduler()
 
         return self._scheduler(engine, name)
+
+
+class LRScheduler(ParamScheduler):
+    """A wrapper class to call `torch.optim.lr_scheduler` objects as `ignite` handlers.
+
+    Args:
+        lr_scheduler (subclass of `torch.optim.lr_scheduler._LRScheduler`): The lr_scheduler object to wrap.
+        save_history (bool, optional): whether to log the parameter values
+            (default=False)
+
+    .. code-block:: python
+
+        from ignite.contrib.handlers.param_scheduler import LRScheduler
+        from torch.optim.lr_scheduler import StepLR
+
+        step_scheduler = StepLR(optimizer, step_size=3, gamma=0.1)
+        scheduler = LRScheduler(step_scheduler)
+
+        trainer.add_event_handler(Events.ITERATION_COMPLETED, scheduler)
+    """
+    def __init__(self, lr_scheduler, save_history=False, **kwds):
+
+        self.lr_scheduler = lr_scheduler
+        super(LRScheduler, self).__init__(
+            optimizer=self.lr_scheduler.optimizer,
+            param_name='lr',
+            save_history=save_history
+        )
+
+    def __call__(self, engine, name=None):
+
+        #
+        # Call the pytorch scheduler step method.
+        #
+        self.lr_scheduler.step(engine.state.epoch)
```

Comment:
I think we should not attach the scheduler to the epochs.
Why can't we simply do 
```python
self.lr_scheduler.last_epoch += 1
```
without any update by pytorch and update params from ParamScheduler side ?